docs/config_file_documentation.txt

```diff
@@ -1,32 +1,29 @@
 This document lists the parameters that can be included in a config file. Required parameters are denoted with an asterisk.
```

Comment:
Maybe better to say "* = required parameter" here at the top or at the bottom. I personally did a ctrl-f for "*" and didn't find this note.